### DISCUSSION TITLE
react-native-searchbox doesn't support Typescript?

### DISCUSSION THREAD

**Discussion by vivijiang** [appbaseio/reactivesearch#1713] [Q&A]
Hi community,

We are building a ReactNative app with TypeScript, now we would like to use react-native-searchbox. But found it doesn't support TypeScript.  Errors attached below. Is there any workaround to get it work? Thanks

Could not find a declaration file for module '@appbaseio/react-native-searchbox'. '/Users/vivi_jiang/Projects/Mango/code/app/mango-react-native/node_modules/@appbaseio/react-native-searchbox/src/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/appbaseio__react-native-searchbox` if it exists or add a new declaration (.d.ts) file containing `declare module '@appbaseio/react-native-searchbox';`

**Comment by siddharthlatest** [comment]
@vivijiang Thanks for bringing attention to this. We will be adding support for Typescript in the Searchbox libraries in the next release for Searchbox libraries: https://github.com/appbaseio/searchbox/issues/111.
